cetloxblock/cetloxblock/cetloxblock.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-

```

Comment:
I noticed we're using 2-space tabs in the Python files here. That's OK, but most people's IDEs are probably configured to use 4-space as that's the norm throughout the community, and what the majority by far is used to.

Not forcing anything here, but I do suggest to re-tab the Python code to be 4-space. Eventually this will become important for quality checking tools too.

If you really want to use 2-space though, then when we configure a quality checking tool like `pep8` or `pylint`, we need to configure it to accept this 2-spacing.

Reply:
tabs changed to 4-space.